src/nav/agents.yml

```diff
@@ -1,1188 +1,2 @@
-      - title: Attributes
-        pages:
-          - title: Enable and disable attributes
-            path: /docs/agents/ruby-agent/attributes/enable-disable-attributes-ruby
-          - title: Ruby agent attributes
-            path: /docs/agents/ruby-agent/attributes/ruby-agent-attributes
-          - title: Attribute examples
-            path: /docs/agents/ruby-agent/attributes/ruby-attribute-examples
-      - title: Instrumented gems
-        pages:
-          - title: Redis instrumentation
-            path: /docs/agents/ruby-agent/instrumented-gems/redis-instrumentation
-          - title: Mongo instrumentation
-            path: /docs/agents/ruby-agent/frameworks/mongo-instrumentation
-          - title: Rack and Metal support
-            path: /docs/agents/ruby-agent/frameworks/rack-metal-support
-          - title: Rack middlewares
-            path: /docs/agents/ruby-agent/instrumented-gems/rack-middlewares
-          - title: Metal controller instrumentation
-            path: /docs/agents/ruby-agent/frameworks/metal-controller-instrumentation
-          - title: Sequel instrumentation
-            path: /docs/agents/ruby-agent/frameworks/sequel-instrumentation
-          - title: Sinatra instrumentation
-            path: /docs/agents/ruby-agent/instrumented-gems/sinatra-instrumentation
-      - title: Troubleshooting
-        pages:
-          - title: No data appears
-            path: /docs/agents/ruby-agent/troubleshooting/no-data-appears-ruby
-          - title: Generating logs
-            path: /docs/agents/ruby-agent/troubleshooting/generating-logs-troubleshooting-ruby
-          - title: Incompatible gems
-            path: /docs/agents/ruby-agent/troubleshooting/incompatible-gems
-          - title: No log file
-            path: /docs/agents/ruby-agent/troubleshooting/no-log-file-ruby
-          - title: Control when the agent starts
-            path: /docs/agents/ruby-agent/troubleshooting/control-when-ruby-agent-starts
-          - title: 'SystemStackError: stack level too deep'
-            path: /docs/agents/ruby-agent/troubleshooting/systemstackerror-stack-level-too-deep
-          - title: Ruby agent audit log
-            path: /docs/agents/ruby-agent/troubleshooting/ruby-agent-audit-log
-          - title: Passenger troubleshooting
-            path: /docs/agents/ruby-agent/troubleshooting/passenger-troubleshooting
-          - title: No data with Unicorn
-            path: /docs/agents/ruby-agent/troubleshooting/no-data-unicorn
-          - title: Update deprecated API calls
-            path: /docs/agents/ruby-agent/troubleshooting/update-deprecated-api-calls
-          - title: Update private API calls to public Tracer API
-            path: /docs/agents/ruby-agent/troubleshooting/update-private-api-calls-public-tracer-api
-          - title: Not installing Grape
-            path: /docs/agents/ruby-agent/troubleshooting/not-installing-new-relic-supported-grape
+path: /docs/apm/agents/intro-to-apm-agents
```

Comment:
I'm not 100% sure, since the builds take forever, but I think this is the line that's causing the build to break. You probably want to delete this file completely!

Reply:
Just realized you added it back in! My understanding of the nav pages is that they also need the `pages` field.